C/plugins/filter/python27/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 2.4.0)
+
+project(python27)
+
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -ggdb")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+# Find source files
+file(GLOB SOURCES *.cpp)
+file(GLOB plugin_common_src "../../../plugins/filter/common/*.cpp" "../../../plugins/common/*.cpp")
+
+# Include header files (first one is python2.7)
+include_directories(/usr/include/python2.7)
+include_directories(include ../../../plugins/filter/common/include ../../../services/common/include ../../../thirdparty/Simple-Web-Server ../../../thirdparty/rapidjson/include ../../../common/include ../../../plugins/common/include)
+
+# Create shared library
+add_library(${PROJECT_NAME} SHARED ${SOURCES} ${plugin_common_src})
+target_link_libraries(${PROJECT_NAME} -lssl -lcrypto)
+
+# Add Python 2.7 library
+target_link_libraries(${PROJECT_NAME} -lpython2.7)
```

Comment:
Do we need to update the README to add another dependency here?

Reply:
Can you make this conditional on the python2.7 code/library being installed so as not to break if we don't have it.

Also please test on Raspbian and Ubuntu

Reply:
Added checks in main CMakeLists.txt and plugin one

Reply:
I don't think we need to update the README file right now for Python dependency.

Python 2.7 filter is now built if libraries are found and in case of errors it's not built and errors are shown via CMake system